task.go

```diff
@@ -28,12 +28,14 @@ import (
 )
 
 type task struct {
-	Deps    []string
-	Links   [][]string
-	Cmds    [][]string
-	Envs    [][]string
-	Accepts [][]string
-	Rejects [][]string
+	Deps     []string
+	Links    [][]string
+    Precmds  [][]string
```

Comment:
Please use tabs, not spaces for indentation, rename `Precmds` to `CmdsPre` and `Postcmds` to `CmdsPost` to conform with naming convention.